index.html

```diff
@@ -55,12 +55,65 @@
     </select>
   </div>
   </main>
-  <footer class="page-footer footer-light transparent">
+  <footer class="page-footer">
```

Comment:
Also, the footer left and right margin seems different from that of coala.io. It would be great if both are consistent.

Reply:
Ok, i will make that soon as soon as I get my lappi in my hands. Not accessible to my laptop for next 5 days ☹️

Reply:
That won't be an issue at all.

Reply:
I guess no, but will check it. By class names, I don’t think so it will adjust the margins

Reply:
@frextrite The `margin` is set to `auto` by a bootstrap class `container`. It isn't being explicitly mentioned in any of the css files for either `coala.io` or `project.coala.io`.

Reply:
In that case, there is nothing that we can do. Giving a go-ahead to the footer design.

Reply:
If everything is ok, can you approve the changes ?